Marlin/ultralcd.cpp

```diff
@@ -1614,15 +1603,11 @@ void kill_screen(const char* lcd_msg) {
      */
     void _lcd_level_bed_homing() {
       if (lcdDrawUpdate) lcd_implementation_drawedit(PSTR(MSG_LEVEL_BED_HOMING), NULL);
+      lcdDrawUpdate = LCDVIEW_CALL_REDRAW_NEXT;
```

Comment:
Maybe, but maybe not. My graphical display is sitting in a box in Portland, Oregon, so I don't have one here to test with, but from my testing today it seems like the overhaul of display handling we did a couple months ago made `LCDVIEW_KEEP_REDRAWING` obsolete.

The latest code in this PR has been patched up so that `PROBE_MANUALLY` now works well on the character display, but it still needs graphical display testing. The separate flags for graphical/character display have been dropped in this PR, so this will also tell us whether they're needed anymore.

If drawing is incorrect on graphical display anywhere in the `LCD_BED_LEVELING` procedures (for `MESH_BED_LEVELING` or `PROBE_MANUALLY`) then I can revisit the drawing flags again.